docs/en-US/component/table-v2.md

```diff
@@ -524,11 +524,12 @@ type RowsRenderedParams = {
 }
 
 type RowSlotProps = {
-  columnIndex: number
-  rowIndex: number
-  data: any
-  key: number | string
+  cells: VNode[]
+  columns: Column<any>[]
+  depth: number
   isScrolling?: boolean | undefined
+  rowData: any
+  rowIndex: number
   style: CSSProperties
```

Comment:
It seems that some properties are missing.
https://github.com/element-plus/element-plus/blob/06e85824df1a7376b198272a7917cf5e855a37fd/packages/components/table-v2/src/table-grid.tsx#L278-L281
https://github.com/element-plus/element-plus/blob/06e85824df1a7376b198272a7917cf5e855a37fd/packages/components/virtual-list/src/types.ts#L185-L192